src/Audit/EnvironmentAnalysis.php

```diff
@@ -64,6 +64,12 @@ public function gather(Sandbox $sandbox) {
       'applicationUuid' => $app['uuid']
     ]));
 
+    if (array_key_exists('remote_admin', $app['flags']) && $app['flags']['remote_admin']) {
```

Comment:
I think we should remove the array_key_exists IF statement because then the various RA policies that would need to check if the ra_settings variable exists or not. The ra_settings variable should exist even if apps don't leverage it, I'm assuming the API would return some sort of response with no settings, or null values, right?

Reply:
The API might return a 404 or 403 which would result in an exception in the application being thrown.

Reply:
Aside from env variables, none of the other api client calls have and IF statement, so the main variable will always be available. Take CSRs for example, if this endpoint doesn't return data, we still get the following available in the policy:

```csrs:
    total: 0
    _links:
        self:
            href: 'https://cloud.acquia.com/api/environments/[UUID]/ssl/csrs'
        parent:
            href: 'https://cloud.acquia.com/api/environments/[UUID]/ssl'
    _embedded:
        items: {  }
```
I feel like having this stubbed out empty response will help prevent policy errors in the future.